setup.py

```diff
@@ -42,6 +42,7 @@
     "proto-plus >= 1.22.0, <2.0.0dev",
     "proto-plus >= 1.22.2, <2.0.0dev; python_version>='3.11'",
     "protobuf>=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
+    "importlib_metadata<=4.13.0; python_version<'3.8'",
```

Comment:
Should we move this to extras, instead of having it as a default dependency?

Also, we should have `importlib_metadata>=1.0.0` and remove the upper bound since this is only used for testing

Reply:
Per conversation, moved it to pandas extras